sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2677,7 +2670,7 @@
           "type": "string"
         },
         "failureReason": {
-          "description": "System genereated.  Ignored at creation.\r\nIf status == failure, this represents a string containing the reason.",
+          "description": "System genereated.  Ignored at creation. If status == failure, this represents a string containing the reason.",
```

Comment:
again, "System genereated.  Ignored at creation." can be removed since we will mark this field as readonly

Reply:
"If status == failure, this represents a string containing the reason." => "Contains the reason for the failure, if a failure occurred"